csrc/mmdeploy/net/CMakeLists.txt

```diff
@@ -46,5 +48,22 @@ if ("tvm" IN_LIST MMDEPLOY_TARGET_BACKENDS)
     add_subdirectory(tvm)
 endif ()
 
-mmdeploy_add_module(${PROJECT_NAME} net_module.cpp)
+if (MMDEPLOY_DYNAMIC_BACKEND)
+    set(_MODULE_STR ${BACKEND_LIB_NAMES})
+    list(TRANSFORM _MODULE_STR REPLACE "(.+)" "\"\\1\"")
+    string(JOIN ",\n        " _MODULE_STR ${_MODULE_STR})
+    set(_MMDEPLOY_DYNAMIC_MODULES ${_MODULE_STR})
```

Comment:
What's this used for?

Reply:
main purpose is to make trt_net;ort_net -> "trt_net";"ort_net"